### ISSUE TITLE
Wappalyzer hangs 

### ISSUE THREAD

**Issue by koencg** [AliasIO/Wappalyzer#2004]
Using version 5.4.3 of the npm package, some sites hang. Here is an example.

node index.js http://www.kikiwomens.be --max-wait=5000 --max-depth=3 --max-urls=3 --recursive=0

**Comment by AliasIO** [comment]
Happens for me too, not every time though. 

Looks like it may be a ZombieJS issue, it hangs at the following debug output:

```
$ export DEBUG=zombie; node index.js http://www.kikiwomens.be --max-wait=5000 --debug=1
```

```
zombie Opened window https://progallery.wix.com/worker.html?cacheKiller=1515644718387&compId=tpaWorker_1133&deviceType=desktop&endpointType=worker&instance=WdQNfzIOfEthuMw4NOe4Pt0idTr6WV6Zp3WBRgh7jnI.eyJpbnN0YW5jZUlkIjoiNDhiZjA0ODYtYWFlOC00NTczLThiYWMtOWMzMGQ2NTI1MWI4IiwiYXBwRGVmSWQiOiIxNDI3MWQ2Zi1iYTYyLWQwNDUtNTQ5Yi1hYjk3MmFlMWY3MGUiLCJzaWduRGF0ZSI6IjIwMTgtMDEtMTFUMDQ6MjU6MTMuNjE5WiIsInVpZCI6bnVsbCwiaXBBbmRQb3J0IjoiMTE4LjEyNy4xMDIuMjMwLzUxODczIiwidmVuZG9yUHJvZHVjdElkIjpudWxsLCJkZW1vTW9kZSI6ZmFsc2UsImFpZCI6IjAyM2ZkZDRiLTYwZDUtNDk1Ny1iODEwLWY3Njg5YTc1NjVkYiIsImJpVG9rZW4iOiI2MmY4MGU0OC03Mzc1LTAzOWUtMTkyYS1hMjVlYTAxNmE4YWEiLCJzaXRlT3duZXJJZCI6IjdkYzMyODU4LTI2YjMtNDhjOC04MWNhLTU2NTg3OTAxMGU4NCJ9&locale=nl&viewMode=site&vsi=317b8fa4-6255-4a98-9506-e9a6cb8e6601 tpaWorker_1133 +5ms
```

**Comment by AliasIO** [comment]
Still not sure what's happening here. It works fine for me 99% of the time, which makes it hard to debug.

As a workaround you can force-quit the script after some time, in `index.js`:

```javascript
setTimeout(() => {
  console.log('force quit');

  process.exit(1);
}, 10000);
```

**Comment by phoenix-schwarz** [comment]
Hangs for me in the AWS Console. Maybe this helps for debugging.

**Comment by gadcam** [comment]
@phoenix-schwarz I do not know if these two issues were linked but what you describe was fixed by #2143.
@koencg  Do you still experience this issue with the current version ?

**Comment by phoenix-schwarz** [comment]
Seems to be fixed. Thanks.
